server/src/main/java/org/elasticsearch/action/support/single/shard/TransportSingleShardAction.java

```diff
@@ -243,7 +249,8 @@ private void perform(@Nullable final Exception currentFailure) {
                             node
                     );
                 }
-                transportService.sendRequest(node, transportShardAction, internalRequest.request(), new TransportResponseHandler<Response>() {
+                transportService.sendRequest(node, transportShardAction, internalRequest.request(), new TransportResponseHandler<Response>()
+                {
```

Comment:
I think moving the whole `new new TransportResponseHandler<Response>() {` onto the next line is a little clearer.

Reply:
+1